rke/k8s_rke_system_images.go

```diff
@@ -5258,6 +5258,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		"v1.19.13-rancher1-1": {
```

Comment:
Add a comment  for v1.19.13 similar to `// Enabled out of band post v2.5.8-patch1` for v1.19.12

Reply:
would be `Enabled out of band post v2.5.9` for v1.19.13